### PR TITLE
ZK-4047: a xsd file for zk.xml

### PR DESCRIPTION
**Pull request by charlesqqq**

### CODE REVIEW

File: zul/src/archive/metainfo/xml/zk.xsd [hunk 1]
```diff
@@ -0,0 +1,390 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns="http://www.zkoss.org/2018/zk"
+		   targetNamespace="http://www.zkoss.org/2018/zk" elementFormDefault="qualified" version="8.6.1.201811121230">
```
Comment (scribetw) [inline zul/src/archive/metainfo/xml/zk.xsd]: https://www.zkoss.org/wiki/ZUML_Reference/ZUML/Namespaces
Name: zk
Namespace: http://www.zkoss.org/2005/zk

Comment (scribetw) [reply zul/src/archive/metainfo/xml/zk.xsd]: http://www.zkoss.org/2005/zk/config

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**